src/components/step-flow/step-flow.style.ts

```diff
@@ -0,0 +1,91 @@
+import styled from "styled-components";
+import Typography from "../typography/typography.component";
+
+const StyledStepFlow = styled.div``;
+
+const StyledStepContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledStepContentText = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledCategoryText = styled(Typography)`
```

Comment:
Comment: we can probably just use `Typography` here directly as it already supports the interface for all this styling